src/DB2Processor.php

```diff
@@ -0,0 +1,48 @@
+<?php namespace JTGrimes\LaravelDB2;
+
+use Illuminate\Database\Query\Builder;
+use Illuminate\Database\Query\Processors\Processor;
+
+class DB2Processor extends Processor
+{
```

Comment:
See if we need these functions or if everything is defined in the abstract class